edb/server/protocol/auth_ext/base.py

```diff
@@ -70,17 +70,15 @@ class ContentType(enum.StrEnum):
     FORM_ENCODED = "application/x-www-form-urlencoded"
 
 
-class OpenIDProvider(BaseProvider):
+class OpenIDConnectProvider(BaseProvider):
     def __init__(
         self,
         name: str,
         issuer_url: str,
         *args: Any,
-        content_type: ContentType = ContentType.JSON,
```

Comment:
The OpenID Connect spec specifies that requests are done via form encoded POST, so I removed this extraneous configuration from the provider since they all support form encoded bodies.